README.md

```diff
@@ -17,6 +17,13 @@ It is available to house members at [proxstar.csh.rit.edu](https://proxstar.csh.
 
 1. [Fork](https://help.github.com/en/articles/fork-a-repo) this repository
   - Optionally create a new [git branch](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell) if your change is more than a small tweak (`git checkout -b BRANCH-NAME-HERE`)
+2. Create a Virtualenv to do your work in.
+```
+mkdir venv
+python3.8 -m venv venv
+source venv/bin/activate
+```
+
```

Comment:
Couldn't they use the dockerfile?

Reply:
They could. This was instructions that I stapled together to run `black` with and didn't document it. I figured that setting something like that up should probably be left to another PR, and I pinky-promise to do that :)